var/spack/repos/builtin/packages/brayns/package.py

```diff
@@ -30,6 +30,7 @@ class Brayns(CMakePackage):
 
     depends_on('assimp', when='+assimp')
     depends_on('brion', when='+brion')
+    depends_on('bbptestdata')
```

Comment:
can we make variant like `test` for this dependency? Idea is people should be able to build brayns even if they don't have access to bbptestdata (which is only internal).